messaging/types/content_topic.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/status-im/status-go/eth-node/types"
+)
+
+const (
+	topicLength = 4
+)
+
+type ContentTopic [topicLength]byte
```

Comment:
This kinda ties us a lot to Waku, right? Though we have no plans to use anything else 😄 

But anyway, I though that Messaging API should encapsulate this logic and Messenger should only operate with "chats"

Reply:
Step by step https://github.com/status-im/status-go/pull/6585#issuecomment-2887558437